js/date-picker/js/lumx.date_picker_directive.js

```diff
@@ -111,6 +145,20 @@ angular.module('lumx.date-picker', [])
             }, 600);
         };
 
+        // month and year toggle activation based on flag
+        // if flag is true, year selection is displayed, else(default) monthly date selection
+        $scope.displayYear = function(flag) {
+            $scope.isYearSelected = flag;
+
+            var $innerListItem = $('.lx-date-picker__years--is-selected');
```

Comment:
Using `angular.element` is the way to go instead of using jquery. Especially since we intend to remove the jquery dependency in the near future.
And instead of searching for all the `.lx-date-picker__years--is-selected`, it would be better to just search inside the`$element`, performance wise but also because we might have multiple date picker on the same page.